### DISCUSSION TITLE
能不能搞个微信群嘛

### DISCUSSION THREAD

**Discussion by xcsweb** [cuixueshe/earthworm#489] [General]
能不能搞个微信群嘛，这样新人上手也好开发

**Comment by PrometheusH** [comment]
确实很难，telegram要美国手机号，我又不想直接用虚拟号，这样容易用着用着被顶走了，就想用google voice来中转一下，原chrome账号被判定活跃地在中国，不给用，重新申请了个chrome账号，可以创建了，google voice又需要美国手机号认证，去找虚拟号，结果google voice好像能识别是虚拟号各种不行，要么就提示这虚拟号也是google voice号不能使用，直接给循环依赖了，创号未半而中道崩殂

**Comment by cuixiaorui** [comment]
有微信群
加我v 说明来意 拉你们
cuixr1314
